Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -266,7 +270,7 @@ public function testWithConsoleHandler()
         $this->assertSame([
             OutputInterface::VERBOSITY_NORMAL => Logger::NOTICE,
             OutputInterface::VERBOSITY_VERBOSE => Logger::INFO,
-            OutputInterface::VERBOSITY_VERY_VERBOSE => 150,
+            OutputInterface::VERBOSITY_VERY_VERBOSE => 200,
```

Comment:
shouldn't this use a constant ?

Reply:
Yes it should but I assume it was testing explicitly with a string value as technically people can use those in the config files. Anyway I just fixed it to have a valid level as 150 does not exist.